heron/common/src/python/argparser.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2016 Twitter. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+''' argparser.py: HERON RC parser support for specifying config level arguments in an RC file.'''
+import argparse
+import collections
+import json
+import os
+import re
+import sys
+import traceback
+import heron.common.src.python.utils.config as config
+import heron.common.src.python.utils.log as log
+
+##########################################################################
+# Run the command
+##########################################################################
+# pylint: disable=invalid-name
+Log = log.Log
```

Comment:
at least in this file, you can do `from import heron.common.src.python.utils.log import Log` since `log` module is not used elsewhere.